Jenkinsfile

```diff
@@ -281,6 +281,16 @@ try {
         }
       }
     },
+    'GPU: Jetson - ARM': {
+      node('mxnetlinux-cpu') {
+        ws('workspace/jetson') {
+          init_git()
+          timeout(time: max_time, unit: 'MINUTES') {
+            sh "python docker_multiarch/tool.py -a jetson"
```

Comment:
We should reconsider our directory structure. At the moment, all CI related dockerfiles are in the ci_build dir, but I also don't think that these dockerfiles should be exclusively for CI.

Reply:
Agree, we should merge these all in the future and get rid of ci_build.sh